pkg/catalog/types.go

```diff
@@ -9,9 +9,8 @@ type Catalog struct {
 }
 
 type Apis struct {
-	Bootstrap            BootstrapApi            `json:"bootstrap"`
-	DataplaneToken       DataplaneTokenApi       `json:"dataplaneToken"` // DEPRECATED: remove in next major version of Kuma
-	MonitoringAssignment MonitoringAssignmentApi `json:"monitoringAssignment"`
+	Bootstrap      BootstrapApi      `json:"bootstrap"`
+	DataplaneToken DataplaneTokenApi `json:"dataplaneToken"` // DEPRECATED: remove in next major version of Kuma
```

Comment:
Is it the right time to remove it?

Reply:
It's not :( If we remove it, Kuma DP 0.x will fail to connect to 1.0 CP